x-pack/plugins/transform/public/app/sections/create_transform/components/source_index_preview/source_index_preview.tsx

```diff
@@ -89,7 +91,9 @@ export const SourceIndexPreview: React.FC<Props> = React.memo(({ indexPattern, q
   const dataGridColumns = indexPatternFields.map(id => {
     const field = indexPattern.fields.getByName(id);
 
-    let schema = 'string';
+    // Built-in values are ['boolean', 'currency', 'datetime', 'numeric', 'json']
+    // To fall back to the default string schema it needs to be undefined.
+    let schema;
```

Comment:
Is sorting not implemented for the source table yet? I can now pick a field(s) to sort by, but the table doesn't update after editing the sort (here I have chosen to sort by `bytes` (third column in the table):

![image](https://user-images.githubusercontent.com/7405507/76210316-c94a5980-61fb-11ea-8c9e-297b2e29299b.png)

Reply:
Pushed a fix in c6b9430 - I thought I implemented that already in the original PR but seems it got lost somehow.